docs/theming.md

```diff
@@ -292,6 +292,8 @@ mwc-button {
 }
 ```
 
+Note: the above selector will not select buttons enclosed in the shadowdom of another component. To target those instances as well, either use the `:root` selector, or append the components containing `mwc-button` in their shadowdoms to the selector (i.e. `mwc-button, my-mwc-button-wrapper { ... }`).
```

Comment:
I think this needs a bit of rewording:

* refrain from using "those, their, that" as it is slightly vague and we need to be as explicit as possible in docs
* caveat with `:root` selector, it doesn't work inside of a shadow root
* I think it might be easier to explain how the custom properties cascade works e.g.

> **Note:** the above selector will not select mwc-buttons enclosed in the shadow roots of other custom elements. CSS Custom Properties (link to MDN) can cascade through shadow root boundaries, but selectors cannot. In order to style mwc-buttons inside of other components' shadow roots, you must select the component that encloses the mwc-button or one of the component's ancestors which will apply the above custom properties.